types/meteor-astronomy/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// Type definitions for meteor-astronomy 2.6
+// Project: https://github.com/jagi/meteor-astronomy/
+// Definitions by: Igor Golovin <https://github.com/Deadly0>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+/// <reference types="meteor" />
+
+type TypeOptionsPrimitives = typeof String | typeof Date | typeof Boolean | typeof Object | typeof Number;
+type TypeOptions = TypeOptionsPrimitives | TypeOptionsPrimitives[] | Class<any> | Enum<any>;
+type MongoQuery = object | string;
+
+interface SaveAndValidateOptions<K> {
+    fields?: K[];
+    stopOnFirstError?: boolean;
+    simulation?: boolean;
+    cast?: boolean;
+}
+
+type SaveAndValidateCallback = (err: any, id: any) => void;
+type RemoveCallback = (err: any, result: any) => void;
+
+interface Validator {
+    type: string;
+    param: any;
+}
+
+type DefaultFunc<T> = () => T;
```

Comment:
Nit: this is only used in one place, I'd suggest inlining it so that nobody takes a dependency on `DefaultFunc`

Reply:
Actually, now that I mention it, is there any way to put these types into a namespace to avoid adding too many global types?